charts/latest/blob-csi-driver/templates/csi-blob-node.yaml

```diff
@@ -99,9 +100,10 @@ spec:
           volumeMounts:
             - name: host-usr
               mountPath: /host/usr
+            - name: host-usr-local
+              mountPath: /host/usr/local
```

Comment:
Is this necessary if you have already `/host/usr` ?

Reply:
Yes, for coreOS, /usr/ and /usr/local are different mount point.

Reply:
I think what @jsafrane mean is that since L102 is already mounting with `/usr`, why not resuse this mount point

Reply:
Becase for coreOS, /usr/local is not a subpath of /usr, it a different mount point, so we have to mount /usr/local and /usr seperately.